### ISSUE TITLE
NeoForged incompatibility

### ISSUE THREAD

**Issue by haruww** [Skidamek/AutoModpack#176]
### Bug description

mod crashes with neoforge

### Steps to reproduce

1. use neoforge 47.1.79 version
2. open minecraft

### Relevant logs

[14:19:16.252] [main/INFO] [AutoModpack/]: AutoModpack prelaunched! took 843ms
14:19:16.261
game
[14:19:16.260] [main/ERROR] [loading.ModSorter/LOADING]: Missing or unsupported mandatory dependencies:
14:19:16.261
game
Mod ID: 'forge', Requested by: 'fabric_resource_loader_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.79'
14:19:16.261
game
Mod ID: 'forge', Requested by: 'fabric_networking_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.79'
14:19:16.261
game
Mod ID: 'forge', Requested by: 'fabric_command_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.79'
14:19:16.261
game
Mod ID: 'forge', Requested by: 'fabric_api_base', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.79'
14:19:16.261
game
Mod ID: 'forge', Requested by: 'fabric_registry_sync_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.1.79'
14:19:17.411
game
Exception in thread "main" java.lang.module.ResolutionException: Modules net.fabricmc.loader and creativecore export package net.fabricmc.api to module json.schema.validator
14:19:17.411
game
at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
14:19:17.411
game
at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
14:19:17.411
game
at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
14:19:17.411
game
at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
14:19:17.411
game
at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
14:19:17.411
game
at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
14:19:17.411
game
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
14:19:17.411
game
at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)

### Minecraft & Mod Loader versions

1.20.1 NeoForged 47.1.79

### Minecraft launcher

Default

### Operating system

Windows10

### AutoModpack version

3.5.2

### Check list

- [x] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by haruww** [comment]
I guess this issue fixed on 3.5.3

**Comment by cfnnit** [comment]
```
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.fabric.FabricLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.quilt.QuiltLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/LanguageProvider/]: [PreloadingTricks/LanguageProvider] Loading settingdust.preloadingtricks.forge.ForgeLanguageProviderCallback
[21:48:44] [main/INFO] [PreloadingTricks/ModSetup/]: [PreloadingTricks/ModSetup] Loading pl.skidam.automodpack_core.Preload
[21:48:44] [main/INFO] [AutoModpack/]: Prelaunching AutoModpack...
[21:48:44] [main/INFO] [AutoModpack/]: Loaded config! took 4ms
[21:48:44] [main/INFO] [AutoModpack/]: Checking if AutoModpack is up-to-date...
[21:48:45] [main/INFO] [AutoModpack/]: Didn't find any updates for AutoModpack! You are on the latest version: 3.5.2
[21:48:45] [main/INFO] [AutoModpack/]: AutoModpack prelaunched! took 430ms
[21:48:45] [main/ERROR] [ne.mi.fm.lo.ModSorter/LOADING]: Missing or unsupported mandatory dependencies:
	Mod ID: 'forge', Requested by: 'fabric_networking_api_v1', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_registry_sync_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_command_api_v2', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_api_base', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
	Mod ID: 'forge', Requested by: 'fabric_resource_loader_v0', Expected range: '[47.1.3,47.1.3]', Actual version: '47.2.1'
Exception in thread "main" java.lang.module.ResolutionException: Modules net.fabricmc.loader and creativecore export package net.fabricmc.api to module lolmha
	at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
	at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
	at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
	at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
	at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
	at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
```
Not yet fixed. sad
